content/language_bindings_and_conformance.tex

```diff
@@ -11,9 +11,10 @@
 implementing the interfaces using macros is strongly discouraged as this could
 severely limit the use of external profiling tools and high-level compiler
 optimizations. An \openshmem program should avoid defining routine names,
-variables, or identifiers with the prefix \shmemprefix (for \Clang{} and
-\Fortran), \shmemprefixC (for \Clang) or with \openshmem \ac{API} names.
+variable names, or identifiers with the prefix \shmemprefix{} (for \Clang{} and
+\Fortran), \shmemprefixC{} (for \Clang) or with \openshmem \ac{API} names.
 
 All \openshmem extension \ac{API}s that are not part of this specification must
 be defined in the \FUNC{shmemx.h} include file. These extensions shall use the
-\FUNC{shmemx\_} prefix for all routine, variable, and constant names.
+\FUNC{shmemx\_} prefix for routine names and variable names and the
```

Comment:
I don't think it's necessary to get this specific about when the lower/uppercase names are used.  Can we change this to: "These extensions shall use the \FUNC{shmemx\_} or \FUNC{SHMEMX_} prefix for all routine, variable, and constant names."